Userland/Libraries/LibWeb/Page/EventHandler.cpp

```diff
@@ -1171,9 +1171,9 @@ EventResult EventHandler::handle_keydown(UIEvents::KeyCode key, u32 modifiers, u
     return EventResult::Accepted;
 }
 
-EventResult EventHandler::handle_keyup(UIEvents::KeyCode key, u32 modifiers, u32 code_point)
+EventResult EventHandler::handle_keyup(UIEvents::KeyCode key, u32 modifiers, u32 code_point, bool repeat)
```

Comment:
Huh, surprised we don't get an "unused parameter" warning here. Should we just not have it here?

Reply:
Ah! spoke to soon, I see we use it later :)

Reply:
That's not very atomic commit-driven of me. Let me fix that up